### PR TITLE
Mapper_frontend Documentation

### LINKED ISSUE #1927: Update the documentation for mapper roles
We have currently developed mapper frontend which has made field mapping experience more smooth experience for mappers on field. But the documentation regarding what is the expected flow for mapper is missing. 

**Solution**

- [ ] We need to add documentation for mapper frontend workflow to provide field mappers and also address any confusion that can arise while they are mapping using FMTM mapnow feature. 
- [ ] Moreover we can also document the possible enhancements that we are thinking on near future to smoothen mappers experience

### PR DESCRIPTION
**Pull request by manjitapandey**
Added step by step guide for mapper frontend

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation
- [ ] 🧑‍💻 Refactor
- [ ] ✅ Test
- [ ] 🤖 Build or CI
- [ ] ❓ Other (please specify)

## Related Issue

Fixes  #1927 

## Describe this PR

Added the step by step guide for mapper frontend workflow

## Alternative Approaches Considered

Did you attempt any other approaches that are not documented in code?

## Review Guide

Notes for the reviewer. How to test this change?

## Checklist before requesting a review

- 📖 Read the FMTM Contributing Guide: <https://github.com/hotosm/fmtm/blob/main/CONTRIBUTING.md>
- 📖 Read the HOT Code of Conduct: <https://docs.hotosm.org/code-of-conduct>
- 👷‍♀️ Create small PRs. In most cases, this will be possible.
- ✅ Provide tests for your changes.
- 📝 Use descriptive commit messages.
- 📗 Update any related documentation and include any relevant screenshots.

### CODE REVIEW

**Comment by manjitapandey** [comment]
I am not able to upload screenshots as mapper frontend isn't accessible at the moment. Will update the documentation directly to add relevant screenshots after the PR is merged.

File: docs/manuals/Mapper_frontend [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Mapper Frontend Documentation
+
+The **Mapper Frontend** is developed to provide a simpler, more intuitive mapping experience.  
+Users can log in directly via URL or access the Mapper Frontend from the project details page.
+
+## How to Start Mapping
+
+Step 1: Access the Mapper Frontend  
+- **Option 1:** Click the **Start Mapping** button on the project details page.  
+- **Option 2:** Go to `fmtm.hotosm.org/mapnow/<project_id>` to open the Mapper Frontend for a specific project.
```
Comment (NSUWAL123) [inline docs/manuals/Mapper_frontend:10]: We have option 3: Redirect to mapper frontend from home page via project card

File: docs/manuals/Mapper_frontend [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  2. Click the project name in the top right corner.  
+  3. Tap the **menu** icon (three horizontal dots).  
+  4. Select **Import QR Code**.  
+  5. Browse to the downloaded QR image and load the project.
+
+For a video guide, click the **info icon** on the Mapper Frontend to watch a video tutorial on how to load the QR code.
+You can load the reference base imagery by clicking on offline mode option (second tab on bottom navigation bar). Choose the layer you want to load and click on *Load layer* to visualise the layer on map.
+
+Step 3: On mapper frontend, lock the feature you want to map by clicking on the feature. This automatically locks the project but mappers in the field won't realise the task is locked. Then, click on 'Go to ODK' to redirect to ODK.
+It prefils the building geometry and surveyors can fill up other details and submit the form.
```
Comment (NSUWAL123) [inline docs/manuals/Mapper_frontend:32]: spelling: prefils ---> prefills

**Review by NSUWAL123 — CHANGES_REQUESTED** [review]
See comments

**Pull request merged**